packages/@glimmer/syntax/tests/parser-node-test.ts

```diff
@@ -470,3 +470,15 @@ test("allow {{undefined}} to be passed as a param", function() {
     b.mustache(b.path('foo'), [b.undefined()])
   ]));
 });
+
+test("Handlebars partial should error", function(assert) {
+  assert.throws(() => {
+    parse("{{> foo}}");
+  }, Error(`Handlebars partials are not supported: "{{> foo" on line "1:0"`));
```

Comment:
Ya, we should say `on line X column Z` or `at L1:C2` (we do the latter in a number of ember's internal errors/warnings).